index.html

```diff
@@ -1,16 +1,109 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <head>
+    <meta charset="utf-8" />
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+      <title>Great Idea!</title></br>
+    <link
+      href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href=".\css\index.css" />
 
-</head>
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+  
+  <header>
+    <div class="head">
+      <div class="head left">
+        <div><a class="items" href="./services.html" >Services</a></div> 
```

Comment:
You need not wrap each of the anchor tags inside a div. You could instead wrap all of them in a nav